pkg/venafi/firefly/connector.go

```diff
@@ -59,7 +60,8 @@ func (c *Connector) RetrieveAvailableSSHTemplates() (response []certificate.SshA
 
 // NewConnector creates a new Firefly Connector object used to communicate with Firefly
 func NewConnector(url string, zone string, verbose bool, trust *x509.CertPool) (*Connector, error) {
-	return &Connector{baseURL: url, zone: zone, verbose: verbose, trust: trust}, nil
+	normalizedUrl := util.NormalizeUrl(url)
```

Comment:
Move this line to a NormalizeUrl function like in other connectors and add a validation for empty url. Just like other connectors, make the NormalizeUrl return an error and use it here in the constructor.

Reply:
It's done